### ISSUE TITLE
Can not load the svg file at browser

### ISSUE THREAD

**Issue by YN-22** [Kaliumhexacyanoferrat/GenHTTP#429]
Hi,

I am coding the application that include the Blazor Wasm UI.
So, I made HTTP server on C# Console Application using this library.
But only svg file was not displayed on the browser.

Processes that occured this problem are follows.
I added following nuget packages

- GenHTTP.Core
- GenHTTP.Modules.StaticWebsites

and added following code.
```　csharp
var tree = ResourceTree.FromDirectory("./wwwroot");
var app = StaticWebsite.From(tree);
Host.Create()
    .Console()
    .Defaults()
    .Handler(app)
    .Run();
```
And then, I placed the web application published by Blazor Wasm project
in the same directory as console application execute file.

```
ParentDirectry
│  ConsoleApp.exe
│
└─wwwroot
    │  index.html
    │
    └─img
            image.svg
```

Launching the server at this situation, I can using Blazor Wasm web application on browser.
But browser cannot be loaded svg file written on razor file.

```　razor
<div>
    <image src="img/image.svg" />
</div>
```
Browser can be loaded ping file written same image tag.
How do I resolve?

Thank you.

**Comment by Kaliumhexacyanoferrat** [comment]
Thanks for reporting, I will have a look into this issue, probably on Monday.

**Comment by Kaliumhexacyanoferrat** [comment]
Has been fixed on master and in bugfix version `8.0.1` of the `GenHTTP.Modules.Basics` nuget package.

**Comment by YN-22** [comment]
Thank you for your swift response.🙇🏻
I can display the svg file!!
